sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - SIG Docs put meaningful effort into growing its contributor and reviewer base in 2021, introducing a shadow program for PR Wrangling as well as dedicating more time to being active via our Slack community channel. This is an ongoing effort to grow our contributor base to a stable number, alongside easing the burden on our small group of reviewers and approvers.
```

Comment:
link to the shadow program docs

Reply:
We don't have a doc per se, Paris. @bradtopol please correct me if I'm mistaken here. We announced it during one of our meetings and asked them to reach out to Brad, if interested. The PR Wranglers list then gets updated with the shadows

Reply:
We can link to the PR Wrangling page itself that has the shadow listing next to each wrangler for the week

Reply:
Is that kind of detail right for an annual report? It'd be better to have a doc about the shadow program generally but right now we don't.

I don't think it's obvious from the PR wrangler list how you become a shadow, which I think is the key thing Paris would want readers to be able to discover.

(instead of adding a link: how about we open and prioritize an issue about adding those contributor docs)?

Reply:
Issue created here: https://github.com/kubernetes/website/issues/31956

Reply:
Added issue mentioned by @natalisucks , hence resolving comment